src/app/body/cart/product-card/product-card.component.html

```diff
@@ -0,0 +1,40 @@
+<div class="product-card">
+  <div class="product-card_right">
+    <a routerLink="/products/{{product.id}}">
+      <img class="product-card_image" [src]="product.imgUrl" alt="{{product.category}}">
+    </a>
+    <button class="openSans"><img src="assets/heart-red.svg" alt="">Wishlist</button>
+    <button (click)="deleteProductFromCart(product)" class="openSans"><img src="assets/delete.svg" alt="">Remove</button>
+  </div>
+  <div class="product-card_content">
+    <div class="product-card_title">{{product.title}}</div>
+    <div class="product-card_info openSans">
+      <div class="product-card_info-item">
+        <span>Farm:</span><span class="black">{{product.farm}}</span>
+      </div>
+      <div class="product-card_info-item">
+        <span>Freshness:</span><span class="black">{{product.freshness}}</span>
+      </div>
+      <div class="product-card_rating">
+        <img *ngFor="let star of productRating" [src]="'assets/' + star + '.svg'" alt="star">
+      </div>
+    </div>
+    <div class="product-card_price">
+      <div class="price_calc">{{product.count}} x {{product.price}} USD</div>
+      <div class="price green">{{(product.price * product.count).toFixed(2)}} USD</div>
+      <div class="count">
+        <input (keydown)="onCountChange()" type="number" [(ngModel)]="product.count"/>
```

Comment:
Why you don't use reactive forms here?

Reply:
I don't know, it shoudn't be there at all)
fixed